sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/HierarchiesClient.cs

```diff
@@ -40,6 +40,16 @@ internal HierarchiesClient(TimeSeriesHierarchiesRestClient hierarchiesRestClient
         /// </summary>
         /// <param name="cancellationToken">The cancellation token.</param>
         /// <returns>The pageable list <see cref="AsyncPageable{TimeSeriesHierarchy}"/> of Time Series hierarchies with the http response.</returns>
+        /// <example>
+        /// <code snippet="Snippet:TimeSeriesInsightsSampleGetAllHierarchies">
+        /// // Get all Time Series hierarchies in the environment
```

Comment:
extra //

Reply:
That's coming from a sample snippet, comments in the snippet will look like this which is fine.